### PR TITLE
Add January 2023 roundup

### PR DESCRIPTION
**Pull request by saragibby**
Adds the January 2023 roundup along with the remaining monthly images.

image

### CODE REVIEW

File: www/src/pages/thismonth.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: This Month In SWA
 Welcome to `This Month In SWA` - a monthly roundup of [Azure Static Web Apps](https://docs.microsoft.com/en-us/azure/static-web-apps/overview) news and updates from the Product Team, Cloud & Student Advocacy - and from you, our amazing Developer Community. 
 
 [**Click here to read the latest issue**
-![](../../static/img/png/roundup/oct.png)](/blog/roundup-2022-oct)
+![](../../static/img/png/roundup/nov.png)](/blog/roundup-2022-nov)
```
Comment (saragibby) [inline www/src/pages/thismonth.md:10]: Will update to point to January with separate PR after this is merged so docusaurus doesn't see broken link.

File: www/src/pages/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ function HomepageHeader() {
         <h1 className="hero__title">{siteConfig.title}</h1>
         <p className="hero__subtitle">{siteConfig.tagline}</p>
         */}
-        <h3> #ThisMonthInSWA - Nov 2022! </h3>
+        <h3> #ThisMonthInSWA - Jan 2023! </h3>
         <Link to="blog/roundup-2022-nov">
```
Comment (saragibby) [inline www/src/pages/index.js:23]: Will update this with separate follow up PR once this is merges to avoid the broken link with docusaurus.

**Review by Reshmi-Sriram — COMMENTED** [review]
ah my bad, reviewed the changes now. So sorry for the late reply!

File: www/blog/2023-01-23-thismonth/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+:::
+
+* **Dec 31** | `dev.to` -  [Deep linking with Azure Static Web Apps and Easy Auth](https://dev.to/johnnyreilly/deep-linking-with-azure-static-web-apps-and-easy-auth-26in) by _John Reilly_ Dig in on deep linking and authentication within Azure Static Web Apps. What each is, and how to implement together for your web app. 
+* **Jan 10** | `dotnetthoughts.net` -  [Implementing Authentication in Azure Static Web Apps](https://dotnetthoughts.net/implementing-authentication-in-static-web-apps-part1/) by _Anuraj Parameswaran_ In this two part series, explore creating and running an app on the Azure Static Web App free plan and standard plans, along with setting up authentication for your app. 
+* **Jan 21** | `youtube.com` -  [Azure Static Web Apps for the Enterprise](https://azureday.community/en-live-from-the-netherlands/azure-static-web-apps-for-the-enterprise/) by _Stacy Cashmore_ Azure Static Web Apps is enterprise ready, and Stacy shows you the steps from infrastruction to authencation on how you can utilize Static Web Apps for your organization.
```
[the remark this answers is not in the document]
Comment (saragibby) [reply www/blog/2023-01-23-thismonth/index.md:65]: Thank you for catching!

File: www/blog/2023-01-23-thismonth/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+---
+
+## Dev Resources
+
+_This section covers content, events and code samples from Microsoft authors. Check them out for relevant learning resources and best practices._
+
+* **Dec 6** | `learn.microsoft.com` -  [Publish a Blazor WebAssembly app and .NET API with Azure Static Web Apps](https://learn.microsoft.com/en-us/shows/learn-live/modern-web-development-net6-ep04-publish-blazor-webassembly-net-api-azure-static-web-apps/?WT.mc_id=academic-86055-sagibbon) by _Jon Galloway and James Montemagno_ Follow along with this Learn Live seesions as Jon and James walk you through how to automatically deploy and update a Blazor web application and its API from a GitHub repository. [**Get started**](https://learn.microsoft.com/en-us/training/modules/publish-app-service-static-web-app-api-dotnet/?WT.mc_id=academic-86055-sagibbon) with the accompanying Learn Module.
```
[the remark this answers is not in the document]
Comment (saragibby) [reply www/blog/2023-01-23-thismonth/index.md:51]: good catch!

**Pull request merged**